app/src/androidTest/java/org/kiwix/kiwixmobile/help/HelpRobot.kt

```diff
@@ -66,26 +66,6 @@ class HelpRobot : BaseRobot() {
     )
   }
 
-  fun clickOnLargeZimFiles() {
```

Comment:
this will now cause a compilation error because HelpActivityTest won't be able to find these methods

Reply:
@macgills Yes, sorry, a bit much of try & error here.

Reply:
If you were in android studio the compilation errors would lead you to the specific lines. You have to be in the androidTest sourceSet for it to start figuring stuff out though